code/game/area/areas/station.dm

```diff
@@ -546,6 +550,10 @@
 	name = "\improper Cafeteria"
 	icon_state = "cafeteria"
 
+/area/station/service/barber
+	name = "\improper Barber"
+	icon_state = "cafeteria" // sue me
```

Comment:
![ezgif-5-37d26fa7aa](https://user-images.githubusercontent.com/34697715/179839877-df2f1456-945f-49ab-bacc-beb071e85af7.gif)

It's a six letter one-word area, it would take probably no more two minutes to crank out using another six-letter area as a template, using the `fonts` icon_state as a reference, and then adjusting the colors. You can even do it in DM, it's one of the simpler things that can be done.

Reply:
No worries, didn't realize it was too flashy.

Reply:
> No worries, didn't realize it was too flashy. Nothing should be touching areas_station.dmi right now.

i lied just merge master